### PR TITLE
Patch u6 implementation

### PR DESCRIPTION
**Pull request by SpencerSharkey**
Implements unknown6 via [node-pogo-signature](https://github.com/SpencerSharkey/node-pogo-signature)'s builder utils

### CODE REVIEW

**Comment by cyraxx** [comment]
Dude, awesome, thanks!
Was just now almost finished with my implementation too haha. Yours might be more polished though. I'll give it a look and see if I'll somehow merge the two together or just go with yours.

File: package.json [hunk 1]
```diff
@@ -32,8 +32,9 @@
     "gpsoauthnode": "0.0.5",
     "long": "^3.2.0",
     "node-pogo-protos": "^1.2.0",
-    "request": "^2.74.0",
-    "s2-geometry": "^1.2.7"
+    "request": "^2.73.0",
```
Comment (ghost) [inline package.json:35]: We just updated request to 2.74.0.

Comment (cyraxx) [inline package.json:35]: Looks like I'll manually merge this PR and my WIP implementation into one anyway (while giving @SpencerSharkey credit, of course) so don't worry about it.

**Comment by SpencerSharkey** [comment]
I wasn't totally sure how you'd want the proto messages built, the lib can produce a full unknown6 message but I used the direct encrypt route here.

also, the native C module is being drop-in replaced with a js native method soon... no more building.

**Pull request closed without merging**